api/proto/config/devconfig.proto

```diff
@@ -83,6 +87,9 @@ message EdgeDevConfig {
   // if we set new epoch, EVE sends all info messages to controller
   // it captures when a new controller takes over and needs all the info be resent
   int64 controller_epoch = 25;
+
+  // Active Baseos Config
+  BaseOSConfig baseos = 26;        // BaseOSImage config block
```

Comment:
And then this become a single BaseOS (not BaseOsConfig)

Reply:
As I mentioned above - was taking the approach of evolving BaseOSConfig as opposed to creating new structure.